pytest_pootle/fixtures/models/storefs.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+from collections import OrderedDict
+
+import pytest
+
+
+TRANSLATION_PATHS = OrderedDict(
+    [("default", "gnu_style/po/<language_code>.<ext>"),
+     ("subdir1",
+      "gnu_style_named_folders/po-<filename>/<language_code>.<ext>"),
+     ("subdir2",
+      "gnu_style_named_files/po/<filename>-<language_code>.<ext>"),
+     ("subdir3",
+      "non_gnu_style/locales/<language_code>/<dir_path>/<filename>.<ext>")])
+
+
+@pytest.fixture
+def tp0_store(settings, tp0, tmpdir):
+    from pootle_config.utils import ObjectConfig
+
+    from .store import _require_store
+
+    repo_path = os.path.join(str(tmpdir), "__src__")
+    os.mkdir(repo_path)
+    conf = ObjectConfig(tp0.project)
+    settings.POOTLE_FS_PATH = str(tmpdir)
```

Comment:
actually - ill probs add both, and use the pootle_fs_path fixture to set the setting